packages/paste-rich-text/tsconfig.json

```
@@ -1,8 +1,9 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "sourceRoot": "./lib",
+    "rootDir": "./lib",
```

Biome (2.4.14)

[error] The key rootDir was already declared.

(lint/suspicious/noDuplicateObjectKeys)